config-linux.md

```diff
@@ -37,3 +37,542 @@ The following devices and filesystems MUST be made available in each application
 | /dev/pts     | [devpts](https://www.kernel.org/doc/Documentation/filesystems/devpts.txt)  | |
 | /dev/ptmx    | [device](https://www.kernel.org/doc/Documentation/filesystems/devpts.txt)  | Bind-mount or symlink of /dev/pts/ptmx |
 | /dev/shm     | [tmpfs](https://www.kernel.org/doc/Documentation/filesystems/tmpfs.txt)    | |
+
+## Namespaces
+
+A namespace wraps a global system resource in an abstraction that makes it appear to the processes within the namespace that they have their own isolated instance of the global resource.
+Changes to the global resource are visible to other processes that are members of the namespace, but are invisible to other processes.
+For more information, see [the man page](http://man7.org/linux/man-pages/man7/namespaces.7.html).
+
+Namespaces are specified as an array of entries inside the `namespaces` root field.
```

Comment:
On Thu, Jan 21, 2016 at 11:24:40AM -0800, Doug Davis wrote:
> > +Namespaces are specified as an array of entries inside the `namespaces` root field.
> 
> Since the word "root" appeared in the old version of the text but
> "namespaces" wasn't a root field in the old version, it was below a
> "linux" field, I'm wondering if the "linux" field is removed as part
> of this?

I'm not intentionally changing any linux-ness.  That wording is from
2ce2c866 (runtime: config: linux: add cgroups information, 2015-09-18,
#199).  I agree that the “root” reference is confusing, and would like
to see it go away, but think it's orthogonal to this commit (which is
just copy/pasting this text), because it isn't impacted by the
config/runtime reunification.

[1]: https://github.com/opencontainers/specs/blob/v0.2.0/runtime-config-linux.md#namespaces